packages/site/pages/install.mdx

```diff
@@ -1,5 +1,3 @@
-import React from 'react'
-
 import Badge from '@pluralsight/ps-design-system-badge'
```

Comment:
Looks great.  Makes on-off pages like this nice.  I'd love to see what a conversion might look like (if desirable at all) of one of the component doc pages.

Reply:
Can we do partial/includes with this to make component pages easier? Looks simpler overall which is better.